### PR TITLE
Removed used of deprecated forced_unwind

### PR DESCRIPTION
**Pull request by juanalday**
*Issue number of the reported bug or feature request: #157*

**Describe your changes**
It replaces the (removed) used of boost::coroutine2::detail::forced_unwind in a catch with boost::context

**Testing performed**
Just a basic compilation.

**Additional context**
Details described in issue #157

### CODE REVIEW

**Comment by juanalday** [comment]
To @safaruqi for review

**Review by arosenzweig3 — CHANGES_REQUESTED** [review]

File: quantum/util/impl/quantum_sequencer_experimental_impl.h [hunk 1]
```diff
@@ -187,7 +188,7 @@ Sequencer<SequenceKey, Hash, KeyEqual, Allocator>::executePending(
     {
         rc = task->_func(ctx);
     }
-    catch(const boost::coroutines2::detail::forced_unwind&)
+    catch(const boost::context::detail::forced_unwind&)
```
[the remark this answers is not in the document]
Comment (juanalday) [reply quantum/util/impl/quantum_sequencer_experimental_impl.h:191]: I believe the only reference in that file is in a comment: 
```
    //NOTE: 'boost::coroutines2::detail::forced_unwind' is deprecated
    using CoroutineStackUnwind = boost::context::detail::forced_unwind;
```

[the remark this answers is not in the document]
Comment (arosenzweig3) [reply quantum/util/impl/quantum_sequencer_experimental_impl.h:191]: Ah, apologies, I mis-read and believed the using directive was pointing to the deprecated exception.

**Review by arosenzweig3 — APPROVED** [review]
Looks good.

**Pull request closed without merging**